src/components/plots/AxisBars.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import * as THREE from 'three';
+import { useGlobalStore } from '@/utils/GlobalStates';
+import { useShallow } from 'zustand/shallow';
+import { Edges } from '@react-three/drei'
+
+function Sides({shape, width, thickness=0.01, rotation=new THREE.Vector3(0, 0, 0)} : {shape : THREE.Vector2, width : number, thickness? : number, rotation? : THREE.Vector3}){
+
+    return (
+        <>
+        <group rotation={rotation.toArray()}>        
+            <group position={[0,width/2,0]} >
+                <mesh position={[shape.y/2, 0, 0]}>
```

Comment:
see how I did the previous version: https://github.com/EarthyScience/Browzarr/blob/4be42b21901be70315344bb6a14806b2c4cb219d/src/components/FrameBoxed.tsx its a combo of `mesh` and `LineSegments`, and I think it looks pretty crisp.